src/Workspaces/Core/Portable/FindSymbols/SymbolFinder.FindReferencesServerCallback.cs

```diff
@@ -34,7 +36,6 @@ internal sealed class FindReferencesServerCallback : IEqualityComparer<Serializa
                 _solution = solution;
                 _progress = progress;
                 _cancellationToken = cancellationToken;
-                _definitionMap = new Dictionary<SerializableSymbolAndProjectId, ISymbol>(this);
```

Comment:
we don't need to pass `this` as the equality comparer to this dictionary.  SerializableSymbolGroup already supprots teh right semantics herre.